pkg/controllers/routing/pbr.go

```diff
@@ -39,6 +39,8 @@ func (nrc *NetworkRoutingController) disablePolicyBasedRouting() error {
 		return fmt.Errorf("failed to update rt_tables file: %s", err)
 	}
 
+	/*
```

Comment:
Is this a permanent change _or_ part of a larger (staged) fix ?

Reply:
It's a permanent change on our end until at least we are able to use overlay with kube-router if/when they fix the MTU issue.

Reply:
👍🏼